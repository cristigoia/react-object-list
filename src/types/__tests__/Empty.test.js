import React from 'react'
import {
  snapshotTest,
  configureEnzymeAdapter,
} from 'utils/tests'
import { Empty } from '../'


describe('Empty type', () => {
  it('snapshot test', () => {
    const testValues = [
      {},
      {emptyText: 'test'},
    ]
    testValues.forEach(testValue => {
      snapshotTest(<Empty {...testValue} />)
    })
  })
})
