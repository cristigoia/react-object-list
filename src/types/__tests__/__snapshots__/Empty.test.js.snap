// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty type snapshot test 1`] = `
<span
  className="text-muted"
>
  -
</span>
`;

exports[`Empty type snapshot test 2`] = `
<span
  className="text-muted"
>
  test
</span>
`;
